playbooks/tofino/setup_nodes-lab_trial.yml

```diff
@@ -12,7 +12,6 @@
 # limitations under the License.
 ---
 #Import playbook
```

Comment:
How come we are no longer making a call to setup_hcp.yml or is this something that you're going to change in your next PR?